config/routes.rb

```diff
@@ -9,9 +9,11 @@
     delete '/products' => 'products#destroy'
   end
 
-  resources :cart_items, only: %i[create update]
+  get 'cart', to: 'carts#show'
```

Comment:
maybe it's better to use `resource :cart` here

Reply:
`resource :cart` asks `CartsController`, not `CartController`, so I left it as it is

Reply:
you can specify controller name
https://stackoverflow.com/questions/5112330/is-it-the-rails-way-to-use-singular-or-plural-controller-names